netmiko/cisco/cisco_vxr_ssh.py

```diff
@@ -0,0 +1,634 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+import re
+import time
+import logging
+import os
+import paramiko
+import warnings
+import socket
+
+from netmiko.cisco.cisco_xr import CiscoXrSSH
+from netmiko.netmiko_globals import BACKSPACE_CHAR
+from netmiko.utilities import get_structured_data
+from netmiko.cafy_custom_exceptions import SessionDownException, PromptNotFoundException, PatternNotFoundException
+from netmiko.cafy_custom_exceptions import ConfigCommitError, ConfigModeEnterError, ConfigModeExitError, LOOP_DELAY
+
+work_dir = os.getenv('CAFYKIT_WORK_DIR')
+if work_dir:
+    formatter = logging.Formatter(fmt='%(asctime)s %(levelname)-8s %(message)s',
+                                  datefmt='%Y-%m-%d %H:%M:%S')
+    paramiko_log_folder = os.path.join(work_dir, 'test_paramiko.log')
+    paramiko.util.log_to_file(paramiko_log_folder, level="DEBUG")
+    fh = logging.FileHandler(paramiko_log_folder)
+    fh.setFormatter(formatter)
+    log = logging.getLogger("netmiko")
+    log.addHandler(fh)
+    log.setLevel(logging.DEBUG)
+    log.addHandler(fh)
+else:
+    from netmiko import log
+
+DELAY_FACTOR_DEPR_SIMPLE_MSG = """\n
+Delay Factor is not used in Cisco VXR SSH Netmiko Library.
+Netmiko 4.x and later has deprecated the use of delay_factor.
+You should remove any use of delay_factor=x from this method call.\n"""
+
+MAX_LOOPS_DEPR_SIMPLE_MSG = """\n
+Max Loops is not used in Cisco VXR SSH Netmiko Library.
+Netmiko 4.x and later has deprecated the use of max_loops.
+You should remove any use of max_loops=x from this method call.\n"""
+
+
+class CiscoVxrSSH(CiscoXrSSH):
+    """
+    CiscoVxrSSH is based of CiscoXrSSH -- CiscoBaseConnection
+    """
+
+    def __init__(self, **kwargs):
+        """Constructor
+        """
+        # 30 minutes
+        self.max_read_timeout = kwargs.get('max_read_timeout', 1800)
```

Comment:
remove this completely and only use read_timeout. nobody except cafykit connection library passes this, so change there as well

Reply:
done.